integrated_channels/moodle/transmitters/content_metadata.py

```diff
@@ -31,7 +34,7 @@ def __init__(self, enterprise_configuration, client=MoodleAPIClient):
             client=client
         )
 
-    def _transmit_create(self, channel_metadata_item_map, chunk_size=None):  # pylint: disable=arguments-differ
+    def _transmit_create(self, channel_metadata_item_map, count=0):  # pylint: disable=arguments-differ
```

Comment:
note: if this looks fine, I need to implement this for the rest of the `_transmit_*` classes

Reply:
also I do still need chunk size damnit. I forgot why that was there.